pytest_mh/_private/plugin.py

```diff
@@ -63,6 +64,8 @@ def __init__(self, pytest_config: pytest.Config) -> None:
         self.mh_collect_artifacts: MultihostArtifactsMode = pytest_config.getoption("mh_collect_artifacts")
         self.mh_artifacts_dir: Path = Path(pytest_config.getoption("mh_artifacts_dir"))
         self.mh_compress_artifacts: bool = pytest_config.getoption("mh_compress_artifacts")
+        self.mh_ignore_preferred_topology: bool = pytest_config.getoption("mh_ignore_preferred_topology")
+        self.mh_preferred_topology: str = pytest_config.getoption("mh_preferred_topology")
```

Comment:
What is the use case?

As I understand it, it does not fit well into the main use case, which is to run test only on certain topology by default. We want to decide, what the preferred topology for each test is. Two tests may have different preferred topologies, therefore overriding this setting for all tests to run against one specific topology does not really make sense to me. I would just remove it, with this understanding.

Reply:
I didn't think about that, hrmm.... I intended it for debugging, ad hoc runs, I suppose you would disable preferred topologies and just set the topology marker via the cli to get the result I want. Okay, I'll remove it.